src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -27,6 +27,7 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response)
     throws IOException, ServletException {
+    request.getSession().removeAttribute("user");
```

Comment:
did you put this here just to be safe? otherwise I don't think its needed.

Reply:
I thought I should log user out when they tried to register although I never show the register button when a user is logged in.

Reply:
So yeah. Be safe.